userd/launcher_test.go

```diff
@@ -28,6 +30,8 @@ import (
 	"github.com/snapcore/snapd/userd"
 )
 
+func Test(t *testing.T) { TestingT(t) }
```

Comment:
:flushed:

Reply:
Heh, go check could say something like "OMG PANIC" there's a module "foo" that has TestBar function taking a check.C as argument but there's no TestingT integration.

Reply:
*cough* yes *cough* sorry for this :(